parts/apicheck.pl

```diff
@@ -181,11 +183,26 @@
 static char  *VARarg2;
 static double VARarg3;
 
-#if defined(PERL_BCDVERSION) && (PERL_BCDVERSION < 0x5009005)
+#if defined(D_PPP_BCDVERSION) && (D_PPP_BCDVERSION < 0x5009005)
 /* needed to make PL_parser apicheck work */
 typedef void yy_parser;
 #endif
 
+/* Handle both 5.x.y and 7.x.y and up
```

Comment:
should we also handle cases where PERL_VERSION is not defined and we have to use PATCHLEVEL and SUBVERSION?

Reply:
There's actually no need to as those happen only in very old perls, and this file is not run except from the most recent perl.